src/constants/pageRotation.js

```diff
@@ -0,0 +1,6 @@
+export const PageRotation = {
+  e_0: window.CoreControls.PageRotation.e_90,
+  e_90: window.CoreControls.PageRotation.e_90,
+  e_180: window.CoreControls.PageRotation.e_180,
+  e_270: window.CoreControls.PageRotation.e_270,
```

Comment:
Hmm... I feel like we can just do `window.CoreControls.PageRotation` in our components. No need to wrap another "constant layer" over it?

Reply:
I have removed this constant